build-system/pr-check.js

```diff
@@ -327,7 +327,7 @@ const command = {
     timedExecOrDie('gulp css');
   },
   buildRuntime: function() {
-    timedExecOrDie('gulp build');
+    timedExecOrDie('gulp build --fortesting');
```

Comment:
Add a new row for this command to `contributing/TESTING.md`, explaining that it sets `test` in `AMP_CONFIG` to true.

Reply:
Done